collect_app/src/main/java/org/odk/collect/android/adapters/FileArrayAdapter.java

```diff
@@ -103,4 +105,22 @@ public View getView(int position, View convertView, ViewGroup parent) {
         }
         return v;
     }
+    @Override
+    public boolean isEnabled(int position) {
+        if (areAllItemsEnabled()) {
+            return true;
+        }
+        if(!enabled){
+            return false;
+        }
+        return true;
+    }
+    @Override
+    public boolean areAllItemsEnabled() {
+        return enabled;
+    }
+
+    public void enableAdapterClick(boolean enabled) {
```

Comment:
And I think if you just need the method locally, making it private would tighten the code.